packages/core/src/components/Button/Button.scss

```diff
@@ -56,18 +60,19 @@
   padding: $spacer-half $spacer-1;
 }
 
-.ds-c-button--primary,
-.ds-c-button--primary:visited {
-  background-color: $button-primary-bg;
+.ds-c-button--transparent,
```

Comment:
This whole section is a terrible diff, basically I moved the css for `.ds-c-button--primary`, danger and success to the end of the file so that it would override inverse styles